themes/startwords/assets/scss/print.scss

```diff
@@ -13,32 +13,31 @@
   }
 
   @top-right {
-    content: element(page-header);
+    content: flow(page-header);
```

Comment:
what does `flow` do?

Reply:
It's part of the paged media CSS spec - it takes an element out of the regular document and inserts it into a space for a running header or footer.  

Here's the docraptor documentation for it, which is a little more readable & targeted than the full paged media spec https://docraptor.com/documentation/article/1067094-headers-footers

I'm still not sure why I had to switch from `element` to `flow` but this is what they have in their documentation and it worked the way I wanted. (We use this to put the SW logo in the top right corner, DOI in the bottom left, etc.)